src/search/landmarks/landmark_graph.h

```diff
@@ -62,10 +53,14 @@ class LandmarkNode {
         id = new_id;
     }
 
-    bool is_true_in_state(const State &state) const;
-};
+    Landmark &get_landmark() {
+        return landmark;
+    }
```

Comment:
Where do you need the non-const access? In the factories?

Reply:
Yes, there are some places where properties of landmarks are set after they have been created (e.g., is_true_in_goal in the zhu_givan lm factory or the achievers in the relaxed lm factory).

Reply:
I added a TODO to remove this function when working on making the landmark graph constant after creation.